toqito/states/bb84.py

```diff
@@ -18,16 +18,18 @@ def bb84() -> np.ndarray:
 
     Examples
     ==========
-    The BB84 basis states can be obtained in :code:`toqito` as follows.
+    The BB84 basis states can be obtained in :code:`toqito` as follows in the form of a list of
+    arrays.
 
     >>> from toqito.states import bb84
     >>> x = bb84()
-    >>> print(f"|0> = {x[0][0].T}, \n |1> = {x[0][1].T}")  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
-    |0> = [[1. 0.]],
-     |1> = [[0. 1.]]
-    >>> print(f"|+> = {x[1][0].T}, \n |-> = {x[1][1].T}")  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
-    |+> = [[0.70710678 0.70710678]],
-     |-> = [[ 0.70710678 -0.70710678]]
+    >>> x
```

Comment:
We probably don't need to assign this to a variable and can just do `bb84()` instead of `x = bb84()`.